foodb/lib/src/key_value/key_value_get.dart

```diff
@@ -62,26 +62,29 @@ mixin _KeyValueGet on _AbstractKeyValue {
                 rev: d.rev?.toString() ?? 'undefined',
                 error: 'not_found',
                 reason: 'missing'));
```

Comment:
nit / non-blocker: kinda feel like can just add.
```
...
if (entry == null) {
  doc = BulkGetDoc(...)
  results.add(BulkGetIdDocs(id: d.id, docs: [doc]));
  continue;
}
... current code
```

Reply:
with continue statement, it  wont push the doc into result
```
if() {
  ...
  continue;
}
...
results.add(BulkGetIdDocs(id: d.id, docs: [doc])); // this line will be skipped
```
by following couchdb original bulk_get api,
https://docs.couchdb.org/en/stable/api/database/bulk-api.html#db-bulk-get
try to get doc id which doesnt exist in db should return the doc with missing status.
in couchdb example, it is the `'baz'` id sample result